src/org/labkey/test/util/TestDataGenerator.java

```diff
@@ -411,4 +413,22 @@ public static TestDataGenerator createDomain(String containerPath, DomainProps d
 
         return def.getTestDataGenerator(containerPath);
     }
+
+    static public CommandResponse deleteDomain(final String containerPath, final String schema, final String domainName)
+            throws IOException, CommandException
```

Comment:
I usually catch and rethrow `IOException` wrapped in a `RuntimeException`. It generally means something catastrophic has happened (like the server isn't responding) and there isn't much a caller would/could do about it.

Reply:
Modified.  Copied the same try/catch from createDomain.